src/docs/wasm-shipping-checklist.md

```diff
@@ -43,6 +43,7 @@ See the [staging of type reflection](https://crrev.com/c/1771791) as a reference
 
 ## How to ship a WebAssembly Feature
 
-- In [`src/wasm/wasm-feature-flags.h`](https://cs.chromium.org/chromium/src/v8/src/wasm/wasm-feature-flags.h), move the feature flag from the `FOREACH_WASM_STAGING_FEATURE_FLAG` macro list to the `FOREACH_WASM_SHIPPED_FEATURE_FLAG` macro list.
+- In [`src/wasm/wasm-feature-flags.h`](https://source.chromium.org/chromium/chromium/src/+/master:v8/src/wasm/wasm-feature-flags.h), move the feature flag from the `FOREACH_WASM_STAGING_FEATURE_FLAG` macro list to the `FOREACH_WASM_SHIPPED_FEATURE_FLAG` macro list.
+    - Ensure to add a blink CQ bot on the CL to check for [blink web test](https://v8.dev/docs/blink-layout-tests) failures caused by enabling the feature (add this line to the footer of the CL description: `Cq-Include-Trybots: luci.v8.try:v8_linux_blink_rel`).
```

Comment:
Indentation looks odd, is this meant to be a sublist with a single item?

Reply:
Yes. First I only added two dots, but the linter complained and demands four spaces.
We have the same thing in line 14-16 above.

The previous looks good. Do you see a problem with this?

Reply:
Not really, just wanted to make sure this is intentional, because from text itself it doesn't sound like a sub-item of the parent.

Reply:
Oh, I see. Would it help to rewrite the first item as "Create a CL to move ..."?
@gahaas WDYT?